### ISSUE TITLE
cargo install fails 

### ISSUE THREAD

**Issue by dzmitry-lahoda** [osmosis-labs/beaker#120]
```
   Compiling cargo-generate v0.15.2
error[E0283]: type annotations needed
   --> /home/dz/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-generate-0.15.2/src/lib.rs:354:39
    |
354 |         let file_path = search_folder.join(name.borrow());
    |                                       ^^^^      ------ type must be known at this point
    |                                       |
    |                                       cannot infer type of the type parameter `P` declared on the method `join`
    |
    = note: multiple `impl`s satisfying `str: Borrow<_>` found in the following crates: `bstr`, `core`:
            - impl Borrow<bstr::bstr::BStr> for str;
            - impl<T> Borrow<T> for T
              where T: ?Sized;
help: consider specifying the generic argument
    |
354 |         let file_path = search_folder.join::<&Borrowed>(name.borrow());
    |                                           +++++++++++++

For more information about this error, try `rustc --explain E0283`.
error: could not compile `cargo-generate` (lib) due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `beaker v0.1.5`, intermediate artifacts can be found at `/tmp/cargo-installhaoPLv`
dz@pop-os:~/github.com/dzmitry-lahoda-forks/beaker$ 

```

please specify exact rust version to use

**Comment by sundaycoderock** [comment]
no fix for this??? i have the same issue

**Comment by iboss-ptk** [comment]
Should be fixed by `0.1.6`

**Comment by dzmitry-lahoda** [comment]
fixed. thank you
